### PR TITLE
Update-eol-versions-6.0

### PR DESCRIPTION
**Pull request by richlander**

### CODE REVIEW

File: eng/targets/Helix.Common.props [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-    <HelixQueueDebian11>(Debian.11.Amd64.Open)Ubuntu.2004.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-11-helix-amd64</HelixQueueDebian11>
-    <HelixQueueFedora34>(Fedora.34.Amd64.Open)Ubuntu.2004.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-34-helix</HelixQueueFedora34>
-    <HelixQueueMariner>(Mariner)Ubuntu.2004.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:cbl-mariner-1.0-helix</HelixQueueMariner>
-    <HelixQueueArmDebian11>(Debian.11.Arm64.Open)ubuntu.2004.armarch.open@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-11-helix-arm64v8</HelixQueueArmDebian11>
+    <HelixQueueAlpine317>(Alpine.317.Amd64.Open)Ubuntu.2204.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.17-helix-amd64</HelixQueueAlpine314>
+    <HelixQueueDebian11>(Debian.11.Amd64.Open)Ubuntu.2204.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-11-helix-amd64</HelixQueueDebian11>
+    <HelixQueueFedora39>(Fedora.39.Amd64.Open)Ubuntu.2204.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-39-helix</HelixQueueFedora34>
```
Comment (wtgodbe) [inline eng/targets/Helix.Common.props:6]: There is no fedora39 image available yet, see https://github.com/dotnet/aspnetcore/pull/54752/files

Comment (richlander) [reply eng/targets/Helix.Common.props:6]:
```suggestion
    <HelixQueueFedora38>(Fedora.38.Amd64.Open)Ubuntu.2204.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-38-helix</HelixQueueFedora38>
```

File: eng/targets/Helix.Common.props [hunk 2]
```diff
@@ -1,11 +1,11 @@
 <Project>
   <!-- this file is shared between Helix.proj and .csproj files -->
   <PropertyGroup>
-    <HelixQueueAlpine314>(Alpine.314.Amd64.Open)Ubuntu.2004.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.14-helix-amd64</HelixQueueAlpine314>
-    <HelixQueueDebian11>(Debian.11.Amd64.Open)Ubuntu.2004.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-11-helix-amd64</HelixQueueDebian11>
-    <HelixQueueFedora34>(Fedora.34.Amd64.Open)Ubuntu.2004.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-34-helix</HelixQueueFedora34>
-    <HelixQueueMariner>(Mariner)Ubuntu.2004.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:cbl-mariner-1.0-helix</HelixQueueMariner>
-    <HelixQueueArmDebian11>(Debian.11.Arm64.Open)ubuntu.2004.armarch.open@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-11-helix-arm64v8</HelixQueueArmDebian11>
+    <HelixQueueAlpine317>(Alpine.317.Amd64.Open)Ubuntu.2204.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.17-helix-amd64</HelixQueueAlpine314>
```
Comment (richlander) [inline eng/targets/Helix.Common.props:4]:
```suggestion
    <HelixQueueAlpine317>(Alpine.317.Amd64.Open)Ubuntu.2204.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.17-helix-amd64</HelixQueueAlpine317>
```

File: eng/targets/Helix.Common.props [hunk 3]
```diff
@@ -34,9 +34,9 @@
     <HelixAvailableTargetQueue Include="OSX.13.Amd64.Open" Platform="OSX" />
 
     <!-- Containers -->
-    <HelixAvailableTargetQueue Include="$(HelixQueueAlpine314)" Platform="Linux" />
+    <HelixAvailableTargetQueue Include="$(HelixQueueAlpine317)" Platform="Linux" />
     <HelixAvailableTargetQueue Include="$(HelixQueueDebian11)" Platform="Linux" />
-    <HelixAvailableTargetQueue Include="$(HelixQueueFedora34)" Platform="Linux" />
+    <HelixAvailableTargetQueue Include="$(HelixQueueFedora39)" Platform="Linux" />
```
Comment (richlander) [inline eng/targets/Helix.Common.props:39]:
```suggestion
    <HelixAvailableTargetQueue Include="$(HelixQueueFedora38)" Platform="Linux" />
```

File: eng/common/templates-official/jobs/source-build.yml [hunk 4]
```diff
@@ -14,7 +14,7 @@ parameters:
   # This is the default platform provided by Arcade, intended for use by a managed-only repo.
   defaultManagedPlatform:
     name: 'Managed'
-    container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7-3e800f1-20190501005343'
```
Comment (wtgodbe) [inline eng/common/templates-official/jobs/source-build.yml:17]: These files come from Arcade & will be overwritten by the next Arcade update - I recommend making the updates in https://github.com/dotnet/arcade/blob/release/6.0/eng/common/templates/jobs/source-build.yml and https://github.com/dotnet/arcade/blob/release/6.0/eng/common/templates-official/jobs/source-build.yml

**Review by wtgodbe — APPROVED** [review]

**Comment by richlander** [comment]
Thanks for the tip on the common files. I keep doing that!

I have PRs on Arcade waiting to merged.

**Review by wtgodbe — APPROVED** [review]

**Pull request merged**